src/platforms/common/session-replay/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Set Up Session Replay
+sidebar_order: 4000
+supported:
+  - javascript
+  - javascript.angular
+  - javascript.ember
+  - javascript.gatsby
+  - javascript.nextjs
+  - javascript.react
+  - javascript.remix
+  - javascript.svelte
+  - javascript.vue
+description: "Learn how to enable session replay in your app if it is not already set up."
+---
+
+With [session replay](/product/session-replay/), TODO.
+
+<PlatformSection supported={["javascript", "javascript.vue", "java.spring", "react-native"]} notSupported={["java", "go", "android", "ruby", "python", "dotnet", "dart", "native"]}>
+
+## Enable Session Replay
+
+<PlatformContent includePath="session-replay/enable-session-replay" />
+
+</PlatformSection>
+
+## Configure the Sample Rate
```

Comment:
I think we should change the order here a little in the sense that it's most important for users to understand how to set up Replay. So under "Enable Session Replay" I think we should already show both code examples (installing the package and the `Sentry.init` call). 

Below then, we can describe the impact of the sample rates and (maybe later on) the other configuration options. 

Side note (no relevant for this PR): I just saw that our docs for Performance setup have the same structure so we might wanna refactor it a bit there as well.